.gitlab-ci.yml

```
@@ -8,6 +8,8 @@ variables:
   S3_BUCKET_SUBPATH: "dist-packages/debian"
   # APT repo path
   S3_BUCKET_REPO_PATH: "repos/debian"
+  # Scripts folder subpath
+  S3_BUCKET_SCRIPTS_PATH: "scripts"
```

I've been inspecting a bit the bucket. We have a lot of stuff here, and I think is better for the script to share the directory with the repos. Let's change this to
```suggestion
  # Scripts folder subpath
  S3_BUCKET_SCRIPTS_PATH: "repos/scripts"
```